extensions/wikia/MercuryApi/handlers/MercuryApiFilePageHandler.class.php

```diff
@@ -11,8 +11,10 @@ public static function getFileContent( Title $title ) {
 
 		$details = WikiaFileHelper::getMediaDetail( $title );
 		$mediaObject = ArticleAsJson::createMediaObject( $details, $title->getText() );
-		$mediaObject['srcset'] = ArticleAsJson::getSrcset( $mediaObject['url'], $mediaObject['width'] );
-		$mediaObject['thumbnailUrl'] = ArticleAsJson::getThumbnailUrlForWidth( $mediaObject['url'], 340 );
+		if ( $mediaObject['isOgg'] === false ) {
```

Comment:
Are you sure it's the only type we should handle this way?

Reply:
It is the only type we can currently differentiate from others. We'll probably need to do followup ticket to handle other file types like .pdf .odf .woff and so on, but it is more work to do and AFAIK ogg is the most common after image / video

Reply:
👍